### PR TITLE
Metronome Battle: Update Bans

### PR DESCRIPTION
**Pull request by Ridaz**
Don't try to circumvent the rules please, thanks
Banned Berserk Gene, Mawilite, Medichamite
Also, removed the nickname clause

### CODE REVIEW

**Comment by MacChaeger** [comment]
Might want to ban Berserk Gene too.

File: config/formats.js [hunk 1]
```diff
@@ -378,19 +378,18 @@ let Formats = [
 		},
 		ruleset: ['HP Percentage Mod', 'Cancel Mod'],
 		banlist: [
-			'Aegislash', 'Aerodactylite', 'Aggronite', 'Aguav Berry', 'Ampharosite', 'Assault Vest', 'Berry', 'Berry Juice', 'Black Sludge',
+			'Aegislash', 'Aerodactylite', 'Aggronite', 'Aguav Berry', 'Ampharosite', 'Assault Vest', 'Berry', 'Berry Juice', 'Berserk Gene', 'Black Sludge',
 			'Blastoisinite', 'Blazikenite', 'Charizardite X', 'Charizardite Y', 'Cheek Pouch', 'Cursed Body', 'Desolate Land', 'Diancite',
-			'Drizzle ++ Dry Skin', 'Drizzle ++ Rain Dish', 'Doublade', 'Enigma Berry', 'Figy Berry', 'Fur Coat', 'Galladite', 'Garchompite',
+			'Doublade', 'Dry Skin', 'Enigma Berry', 'Figy Berry', 'Fur Coat', 'Galladite', 'Garchompite',
```
Comment (KrisXV) [inline config/formats.js]: Why is the spacing so inconsistent??????

Comment (Marty-D) [reply config/formats.js]: Don't worry! I'm reordering all of this as we speak!

**Comment by Ridaz** [comment]
I forgot about Fluffy

**Comment by Marty-D** [comment]
Do you want to also ban Mawile-Mega instead of Mawilite, etc? Or it doesn't matter?

**Comment by Ridaz** [comment]
It matters. Those Megas are allowed. The abilities Huge Power/Pure Power are banned, not the Pokémons themselves. You can use them directly

File: config/formats.js [hunk 2]
```diff
@@ -378,15 +378,16 @@ let Formats = [
...
-			'Gardevoirite', 'Gold Berry', 'Grassy Surge', 'Gyaradosite', 'Harvest + Rowap Berry', 'Harvest + Jaboca Berry', 'Honedge',
-			'Huge Power', 'Iapapa Berry', 'Ice Body', 'Iron Barbs', 'Kitsunoh', 'Latiasite', 'Latiosite', 'Leftovers', 'Lucarionite',
-			'Mago Berry', 'Mawilite', 'Medichamite', 'Metagrossite', 'Moody', 'Normalium Z', 'Oran Berry', 'Parental Bond', 'Poison Heal', 'Power Construct', 'Pressure',
-			'Primordial Sea', 'Protean', 'Pure Power', 'Rain Dish', 'Rocky Helmet', 'Rough Skin', 'Salamencite', 'Sand Stream', 'Sceptilite', 'Schooling',
-			'Shedinja + Sturdy', 'Sitrus Berry', 'Snow Warning', 'Stamina', 'Steelixite', 'Swampertite', 'Tyranitarite', 'Venusaurite',
-			'Water Absorb', 'Volt Absorb', 'Wiki Berry', 'Wonder Guard',
+			'Aegislash', 'Doublade', 'Honedge', 'Kitsunoh',
+			'Cheek Pouch', 'Cursed Body', 'Desolate Land', 'Dry Skin', 'Fur Coat', 'Grassy Surge', 'Huge Power', 'Ice Body', 'Iron Barbs',
+			'Moody', 'Parental Bond', 'Poison Heal', 'Power Construct', 'Pressure', 'Primordial Sea', 'Protean', 'Pure Power', 'Rain Dish',
+			'Rough Skin', 'Sand Stream', 'Schooling', 'Snow Warning', 'Stamina', 'Water Absorb', 'Volt Absorb', 'Wonder Guard',
```
Comment (urkerab) [inline config/formats.js]: ... although you missed your chance to put Volt Absorb before Water Absorb...

**Pull request merged**